apps/server/src/modules/tool/uc/context-external-tool.uc.ts

```diff
@@ -42,4 +46,43 @@ export class ContextExternalToolUc {
 
 		return promise;
 	}
+
+	async getContextExternalToolsForContext(userId: EntityId, contextType: ToolContextType, contextId: string) {
```

Comment:
Maybe we could combine `contextType` and `contextId` as a unit `context` here?

Reply:
Good idea :) !